### PR TITLE
updated PropertyAccess docs for PR 30536

### PR DESCRIPTION
**Pull request by wuestkamp**
PR: https://github.com/symfony/symfony/pull/30545
Issue: https://github.com/symfony/symfony/issues/30536

### CODE REVIEW

File: components/property_access.rst [hunk 1]
```diff
@@ -168,6 +168,29 @@ getters, this means that you can do something like this::
 
 This will produce: ``He is an author``
 
+Accessing a non existing property path
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (OskarStark) [inline components/property_access.rst]: Please add a blank line afterwards

File: components/property_access.rst [hunk 2]
```diff
@@ -168,6 +168,29 @@ getters, this means that you can do something like this::
...
+        ->getPropertyAccessor();
+
+    class Person
+    {
+        public $name;
+    }
+
+    $person = new Person();
+
+    // instead of throwing an exception the following code returns a null value
```
Comment (OskarStark) [inline components/property_access.rst]: s/a null value/null/

File: reference/configuration/framework.rst [hunk 3]
```diff
@@ -1726,6 +1727,14 @@ throw_exception_on_invalid_index
 When enabled, the ``property_accessor`` service throws an exception when you
 try to access an invalid index of an array.
 
+throw_exception_on_invalid_property_path
+........................................
+
+**type**: ``boolean`` **default**: ``true``
+
```
Comment (OskarStark) [inline reference/configuration/framework.rst]: Please add a versionadded directive to show when this was added

**Review by OskarStark — COMMENTED** [review]
Thank you for your contribution 👍🏻

**Comment by wouterj** [comment]
Thank you @wuestkamp! I've slightly updated your changes in 98dfd5d3d, please let me know if you disagree with something.

**Pull request merged**